diff --git a/dist/esm/index.d.ts b/dist/esm/index.d.ts
index 41d16a98ebe793da4033a0bfc7c24d0055bee45d..87c11d8c4852f4605a45d3be19e2ada8ea066701 100644
--- a/dist/esm/index.d.ts
+++ b/dist/esm/index.d.ts
@@ -44,6 +44,7 @@ export declare const defaults: (def: MinimatchOptions) => typeof minimatch;
 export declare const braceExpand: (pattern: string, options?: MinimatchOptions) => string[];
 export declare const makeRe: (pattern: string, options?: MinimatchOptions) => false | MMRegExp;
 export declare const match: (list: string[], pattern: string, options?: MinimatchOptions) => string[];
+export default minimatch;
 export type MMRegExp = RegExp & {
     _src?: string;
     _glob?: string;
diff --git a/dist/esm/index.js b/dist/esm/index.js
index 84b577b0472cb6cb2fe4d5f2a146867fc5bdd8c5..75163cb3c970711b9035caa9dea203b5cd6ee6ff 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -998,4 +998,5 @@ minimatch.AST = AST;
 minimatch.Minimatch = Minimatch;
 minimatch.escape = escape;
 minimatch.unescape = unescape;
+export default minimatch;
 //# sourceMappingURL=index.js.map
